data/selinux/snappy.fc

```diff
@@ -18,30 +18,38 @@
 
 
 HOME_DIR/snap(/.*)?			gen_context(system_u:object_r:snappy_home_t,s0)
+/root/snap(/.*)?			gen_context(system_u:object_r:snappy_home_t,s0)
```

Comment:
This should not be necessary, since `HOME_DIR` applies to `/root` too.

Reply:
I wasn't sure about that. I recall seeing separate rules for both `HOME_DIR` and `/root` in the core policy.

Reply:
Right here: https://github.com/fedora-selinux/selinux-policy/blob/rawhide/policy/modules/services/ssh.fc

Reply:
Ah, okay.